common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js

```diff
@@ -101,5 +140,10 @@
     if (baseFontSize && parseInt(baseFontSize, 10) !== 16) {
         documentElement.style.fontSize = baseFontSize
     }
+
+    if (hideReaderRevenue()) {
+        docClass += ' hide-reader-revenue';
```

Comment:
`has-contributed` perhaps? could we also provide helper classes along the lines of `hide-on-mobile`? is that useful? `hide-from-paid` maybe? maybe this is muddying the water a bit...